collector/dhcp.go

```diff
@@ -0,0 +1,401 @@
+// +build windows
+
+package collector
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	registerCollector("dhcp", NewDhcpCollector, "DHCP Server")
+}
+
+// A DhcpCollector is a Prometheus collector for both WMI and perflib DHCP metrics
```

Comment:
Nit: As far as I find in a seconds readthrough now, it doesn't actually have any WMI readouts?